glean-core/android/src/test/java/mozilla/telemetry/glean/private/PingTypeTest.kt

```diff
@@ -201,17 +201,17 @@ class PingTypeTest {
         val server = getMockWebServer()
 
         val counter = CounterMetricType(
-            disabled = false,
-            category = "test",
-            lifetime = Lifetime.Ping,
-            name = "counter",
-            sendInPings = listOf("unknown")
+                disabled = false,
```

Comment:
Out of curiosity, why are we changing indentation here and in other places?

Reply:
by accident.